SabreTools.Models/PlayStation4/AppPkgHeader.cs

```diff
@@ -0,0 +1,238 @@
+using System.Runtime.InteropServices;
+
+namespace SabreTools.Models.PlayStation4
+{
+    /// <see href="https://www.psdevwiki.com/ps4/PKG_files"/>
+    [StructLayout(LayoutKind.Sequential)]
+    public class AppPkgHeader
+    {
+        /// <summary>
+        /// 0x7F434E54
+        /// ".CNT"
+        /// </summary>
+        [MarshalAs(UnmanagedType.ByValArray, SizeConst = 4)]
+        public byte[]? Magic;
+
+        /// <summary>
+        /// PKG Type
+        /// </summary>
+        public uint Type;
+
+        /// <summary>
+        /// PKG Unknown Field
+        /// </summary>
+        public uint PKGUnknown;
+
+        /// <summary>
+        /// PKG File count
+        /// </summary>
+        public uint FileCount;
+
+        /// <summary>
+        /// PKG Entry count
+        /// </summary>
+        public uint EntryCount;
+
+        /// <summary>
+        /// SC Entry count
+        /// </summary>
+        public ushort SCEntryCount;
+
+        /// <summary>
+        /// PKG Entry count (duplicated)
+        /// </summary>
+        public ushort EntryCount2;
+
+        /// <summary>
+        /// PKG Table offset

```

Comment:
For fields like this one, please use the comments from the linked wiki page since they're a bit more descriptive. There's a few that are like this.